### ISSUE TITLE
issue not closing via commit with user/repo#id

### ISSUE THREAD

**Issue by ms502040** [gitbucket/gitbucket#2369]
when in commit message `resolve  user/repo#id ` issue not closed only link to user/repo#id in file listing is correctly.

**Comment by takezoe** [comment]
Closing by comment works for only in the same repository.

**Comment by ms502040** [comment]
yes, in same repo working well. but would be nice to support this too: https://help.github.com/en/articles/closing-issues-using-keywords#closing-an-issue-in-a-different-repository   
#issue_number is unique for repo, if you have submodule in git you can have mismatch in closed issues.

**Comment by takezoe** [comment]
I see. I mark this as new feature request. Thanks.
